src/EntryPoints/Rest/SaveMappingsApi.php

```diff
@@ -17,8 +17,10 @@ public function run( string $entityId ): Response {
 		$presenter = WikibaseRdfExtension::getInstance()->newRestSaveMappingsPresenter( $this->getResponseFactory() );
 		$useCase = WikibaseRdfExtension::getInstance()->newSaveMappingsUseCase( $presenter, $this->getUser() );
 
-		$body = is_array( $this->getValidatedBody() ) ? $this->getValidatedBody() : [];
-		$mappings = isset( $body['mappings'] ) ? (array)$body['mappings'] : [];
+		$body = $this->getValidatedBody();
+		$mappings = ( is_array( $body ) && isset( $body['mappings'] ) )
```

Comment:
psalm keeps complaining about the type when it is clear, so I added the error to the baseline